contexts/NotificationsContext.tsx

```diff
@@ -4,28 +4,35 @@ import Notifications from '@/components/Notifications';
 export type TxNotification = {
   id: string;
   hash: `0x${string}`;
-  functionName: string;
+  functionName?: string;
```

Comment:
should always have function name

Reply:
In `useEasyWrite` we use `params.functionName` which has a type of `string | undefined`. I made this change to resolve that issue.